src/Data/Managers/V2/Notification/GetNotifications/getUrl.test.ts

```diff
@@ -9,9 +10,8 @@ it.each`
 `(
   "getUrl returns correct url for agents with filter $filter, currentPage: $currentPage, and pageSize: $pageSize",
```

Comment:
Is it me, or is the test description not correct? It's referring to agents with filters, but we are in the Notifications tests